client/packages/common/src/hooks/useUrlQuery/useUrlQueryParams.ts

```diff
@@ -141,9 +141,9 @@ export const useUrlQueryParams = ({
         : 0,
     first: rowsPerPage,
     sortBy: {
-      key: urlQuery['sort'] ?? '',
-      direction: urlQuery['dir'] ?? 'asc',
-      isDesc: urlQuery['dir'] === 'desc',
+      key: urlQuery['sort'] ?? initialSort?.key ?? '',
```

Comment:
offroad: this bug is adding errors to console. user shouldn't notice, so I didn't create a new issue.
the problem is that when you click a menu item (in this case 'stocktake') the sort key is empty and an error raised - even though the 'initialSort' has been provided.

To check - try develop branch, or undo this change. Click on stocktake in the left hand menu and check console for errors. You'll also see the network request fail with an error.

Reply:
Ohh yeah i remember seeing this error, was going to make an issue, thanks for fixing !